src/pages/ApplicantDetailsPage.tsx

```diff
@@ -0,0 +1,160 @@
+
+  useEffect(() => {
+    setApplicantRating(props.applicant.rating);
+  }, [props.applicant.rating]);
+
+  return (
+    <Box
+      sx={{
+        flex: 1,
+        height: 'calc(100vh - APP_BAR_HEIGHT)',
+        marginTop: APP_BAR_HEIGHT,
+      }}
+    >
+      <Stack sx={{ marginLeft: 3, marginTop: 10, pb: 2 }} direction="column">
+        <Typography variant="h4" fontWeight={'bold'} sx={{ color: '#4d4d4d' }}>
+          Java Developer Backend
+        </Typography>
+        <Typography variant="h6" sx={{ color: '#4d4d4d' }}>
+          IT Department
+        </Typography>
+      </Stack>
+      <Box
+        sx={{
+          marginLeft: 3,
+          marginRight: 3,
+          display: 'flex',
+          flexDirection: 'row',
+          justifyContent: 'space-between',
+          alignItems: 'center',
+        }}
+      >
+        {/** TODO: Replace with backend url that serves the profile picture */}
+        <Avatar
+          src={'https://thispersondoesnotexist.com/'}
+          sx={{ width: 100, height: 100, border: 5, borderColor: '#B4CD93' }}
+        ></Avatar>
+        <Typography
+          variant="h5"
+          sx={{
+            color: '#4d4d4d',
+            marginLeft: '1vw',
+            fontWeight: 'bold',
+            minWidth: '8vw',
+          }}
+        >
+          {props.applicant.firstName + ' ' + props.applicant.lastName}
+        </Typography>
+        <Box
+          sx={{
+            width: '100%',
+            flexDirection: 'row',
+          }}
+        >
+          <Box
+            sx={{
+              flexDirection: 'row',
+              display: 'flex',
+            }}
+          >
+            <LinearProgress
+              variant="determinate"
+              value={linearProgressValue}
+              color="secondary"
+              sx={{
+                height: 10,
+                borderRadius: 5,
+                minWidth: '80%',
+                alignSelf: 'center',
+              }}
+            />
+            <Typography
+              variant="body2"
+              color="text.secondary"
+              sx={{ paddingLeft: '1vw', fontSize: 30 }}
+            >
+              {applicantRating + '/10'}
+            </Typography>
+          </Box>
+        </Box>
+        <Box sx={{ marginRight: '4vw' }}>
+          <Typography variant="h6" sx={{ marginLeft: 1 }}>
+            Attached Documents
+          </Typography>
+          {/** TODO: Add functionality to view pdf in browser */}
+          <Button
+            variant="contained"
+            startIcon={<AttachFileIcon />}
+            color="secondary"
+            disableElevation
+            sx={{ margin: 1 }}
+          >
+            CV
+          </Button>
+          <Button
+            variant="contained"
+            startIcon={<AttachFileIcon />}
+            color="secondary"
+            disableElevation
+            sx={{ margin: 1 }}
+          >
+            Cover Letter
+          </Button>
```

Comment:
I would either comment out or delete the cover letter button since we are just doing the cv for now. If we bring back the cover letter, the buttons should be next to each other not above each other.

Reply:
I disabled the button for the cover letter